roxctl/scanner/generate/generate_test.go

```diff
@@ -0,0 +1,29 @@
+package generate
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/apiparams"
+	"github.com/stackrox/rox/pkg/istioutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestScannerGenerateValidation(t *testing.T) {
```

Comment:
How about changing it into table test

Reply:
As we shortly discussed about this. We will keep it as it is.